install.sh

```diff
@@ -236,49 +236,43 @@ main() {
   check_docker
   check_docker_compose
 
-if [ -n "$SUDO_USER" ]; then
-    current_id=$(id -u "$SUDO_USER")
-else
-    current_id=$(id -u)
-fi
-
-if [ "$current_id" -ne 1000 ]; then
-    INSTALL_TYPE="source"
-    log "Build has been forced because your user ID is not the same as the pre-built images. If you want to use pre-built images, your current user installing reNgine-ng must be 1000." $COLOR_RED
-else
-    if [ $isNonInteractive = false ]; then
-        log "Do you want to build Docker images from source or use pre-built images (recommended)? \nThis saves significant build time but requires good download speeds for it to complete fast." $COLOR_RED
-        log "1) From source" $COLOR_YELLOW
-        log "2) Use pre-built images (default)" $COLOR_YELLOW
-        read -p "Enter your choice (1 or 2, default is 2): " choice
-
-        case $choice in
-            1)
-                INSTALL_TYPE="source"
-                ;;
-            2|"")
-                INSTALL_TYPE="prebuilt"
-                ;;
-            *)
-                log "Invalid choice. Defaulting to pre-built images." $COLOR_RED
-                INSTALL_TYPE="prebuilt"
-                ;;
-        esac
-    fi
-fi
-
-if [ -z "$INSTALL_TYPE" ]; then
-    log "Error: INSTALL_TYPE is not set" $COLOR_RED
-    exit 1
-elif [ "$INSTALL_TYPE" != "prebuilt" ] && [ "$INSTALL_TYPE" != "source" ]; then
-    log "Error: INSTALL_TYPE must be either 'prebuilt' or 'source'" $COLOR_RED
-    exit 1
-fi
+  if [ -n "$SUDO_USER" ]; then
+      current_id=$(id -u "$SUDO_USER")
+  else
+      current_id=$(id -u)
+  fi
 
-log "Selected installation type: $INSTALL_TYPE" $COLOR_CYAN
+  # Install type
+  if [ "$current_id" -ne 1000 ]; then
+      # If the user is not 1000, force source install because pre-built images are not compatible with user > 1000
+      INSTALL_TYPE="source"
+      log "Build has been forced because your user ID is not the same as the pre-built images. If you want to use pre-built images, your current user installing reNgine-ng must be 1000." $COLOR_RED
```

Comment:
Do we need to explain/outline how to do this or do you assume this to be common knowledge (first/primary user of the OS)?

Reply:
I've updated the wiki with steps to do this
![image](https://github.com/user-attachments/assets/cf57246a-be66-40af-93d2-ce3ded35e74c)

Reply:
Great. We should definitely also include this in our changelog though. Maybe add a "breaking changes" header?

Reply:
Already there
![image](https://github.com/user-attachments/assets/0b9214b1-5c72-4018-9208-3b8b9aeadbb3)

Reply:
I will add create a sudo user

Reply:
I mean in the changelog of the release.

Reply:
But what we can also do, is just link to wherever these breaking changes are, if you know what I mean.

Reply:
> I mean in the changelog of the release.

We need to update it :grin: 
https://github.com/Security-Tools-Alliance/rengine-ng/blob/master/CHANGELOG.md
![image](https://github.com/user-attachments/assets/9a199ca4-0b22-4fc8-a9e9-20a5dcf30431)

For the moment, only the Changelog in the wiki is up to date

Reply:
Let's discuss this on Discord how we'll set this up correctly.